ms/MSOper/MSMetaData.cc

```diff
@@ -5117,7 +5132,32 @@ vector<MSMetaData::SpwProperties>  MSMetaData::_getSpwInfo2(
             fdmSpw.insert(i);
         }
         else {
-            tdmSpw.insert(i);
+            const auto spwToPol = _getSpwToPolMap();
+            if (
+                spwToPol[i].size() == 1
+                && spwInfo[i].nchans
+                    * getNumCorrs()[*(spwToPol[i].begin())] > 256
```

Comment:
Am I correct in assuming that `tdmSpw` and `fdmSpw` stand for time and frequency domain spectral window? And what is the magic number 256 here?

Reply:
Yes that's what the acronyms mean. The 256 is the number given to me by the stakeholder who requires this change. There is some complexity in this corner case that requires it. I cannot give much of a detailed explanation beyond that; I'm just the guy who codes the requirements I'm given :) .